platform/darwin/test/MGLExpressionTests.mm

```diff
@@ -380,6 +387,12 @@ - (void)testArithmeticExpressionObject {
         XCTAssertEqualObjects([NSExpression expressionWithFormat:@"1 + 1"].mgl_jsonExpressionObject, jsonExpression);
         XCTAssertEqualObjects([NSExpression expressionWithMGLJSONObject:jsonExpression], expression);
     }
+    {
+        NSExpression *expression = [NSExpression expressionForFunction:@"sum:" arguments:arguments];
+        NSArray *jsonExpression = @[@"+", @1, @1];
+        XCTAssertEqualObjects(expression.mgl_jsonExpressionObject, jsonExpression);
+        XCTAssertEqualObjects([NSExpression expressionWithFormat:@"1 + 1"].mgl_jsonExpressionObject, jsonExpression);
+    }
```

Comment:
Similar tests could be added for `max`, `min`, `mgl_join`

Reply:
👍  Working on getting the tests passing.